src/main/java/ua/softserveinc/tc/validator/TimeValidator.java

```diff
@@ -32,11 +32,17 @@ public void validate(Object target, Errors errors) {
         }
     }
 
-    public boolean validateDate(String date)
-    {
+    public boolean validateDateFormat(String date) {
         return date.matches(ValidationConstants.DATE_REGEX);
     }
 
+    public boolean validateDate(String startTime, String endTime) {
+        if(validateDateFormat(startTime) && validateDateFormat(endTime)) {
+            return startTime.compareTo(endTime) <= 0;
```

Comment:
this logic have been implemented in other function of the time validation, for example if you get the same date it will be return all data from that date.